src/webui/src/components/overview/Progress.tsx

```diff
@@ -19,24 +19,105 @@ interface ProgressProps {
 
 interface ProgressState {
     isShowLogDrawer: boolean;
+    isCalloutVisible?: boolean;
+    isShowSucceedInfo: boolean;
+    info: string;
+    typeInfo: string;
 }
 
+const itemStyles: React.CSSProperties = {
+    height: 50,
+    width: 100
+};
+const theme = getTheme();
+const styles = mergeStyleSets({
+    buttonArea: {
+        verticalAlign: 'top',
+        display: 'inline-block',
+        textAlign: 'center',
+        margin: '0 100px',
+        minWidth: 130,
+        height: 32
+    },
+    callout: {
+        maxWidth: 300
+    },
+    header: {
+        padding: '18px 24px 12px'
+    },
+    title: [
+        theme.fonts.xLarge,
+        {
+            margin: 0,
+            color: theme.palette.neutralPrimary,
+            fontWeight: FontWeights.semilight
+        }
+    ],
+    inner: {
+        height: '100%',
+        padding: '0 24px 20px'
+    },
+    actions: {
+        position: 'relative',
+        marginTop: 20,
+        width: '100%',
+        whiteSpace: 'nowrap'
+    },
+    subtext: [
+        theme.fonts.small,
+        {
+            margin: 0,
+            color: theme.palette.neutralPrimary,
+            fontWeight: FontWeights.semilight
+        }
+    ],
+    link: [
+        theme.fonts.medium,
+        {
+            color: theme.palette.neutralPrimary
+        }
+    ]
+});
+
 class Progressed extends React.Component<ProgressProps, ProgressState> {
+    private _menuButtonElement!: HTMLDivElement | null;
+    private _labelId: string = getId('callout-label');
+    private _descriptionId: string = getId('callout-description');
     constructor(props: ProgressProps) {
         super(props);
         this.state = {
-            isShowLogDrawer: false
+            isShowLogDrawer: false,
+            isCalloutVisible: false,
+            isShowSucceedInfo: false,
+            info: '',
+            typeInfo: 'success'
         };
     }
 
+    hideSucceedInfo = (): void => {
+        this.setState(() => ({ isShowSucceedInfo: false }));
+    }
+    /**
+     * info: message content
+     * typeInfo: message type: success | error...
+     * continuousTime: show time, 2000ms 
+     */
+    getMessageInfo = (info: string, typeInfo: string): void => {
+        this.setState(() => ({
+            info, typeInfo,
+            isShowSucceedInfo: true
+        }));
+        setTimeout(this.hideSucceedInfo, 2000);
+    }
+
     editTrialConcurrency = async (userInput: string): Promise<void> => {
         if (!userInput.match(/^[1-9]\d*$/)) {
-            message.error('Please enter a positive integer!', 2);
+            this.getMessageInfo('Please enter a positive integer!', 'error');
```

Comment:
I think "_get_ message info" is misleading.

Reply:
yeah, will change to `showMessageInfo`
It's OK?